src/payment/payment-method-ids.js

```diff
@@ -1,3 +1,10 @@
+export const BIGCOMMERCE_PAYMENTS_PAYPAL = 'bigcommerce_payments_paypal';
+export const BIGCOMMERCE_PAYMENTS_PAYLATER = 'bigcommerce_payments_paylater';
+export const BIGCOMMERCE_PAYMENTS_CREDIT_CARD = 'bigcommerce_payments_creditcard';
```

Comment:
```suggestion
export const BIGCOMMERCE_PAYMENTS_CREDIT_CARDS = 'bigcommerce_payments_creditcards';
```

Reply:
Thanks. I saw the difference between PPCP and BCP CC provider id naming and went to the team to double check. So this change where on my radar.